eng/tools/generate-static-index/service-mapper.json

```diff
@@ -101,7 +101,7 @@
   "sqlvirtualmachine": "SQL Virtual Machine",
   "storage": "Storage",
   "storageimportexport": "Storage Import Export",
-  "storagesync": "Storage Sync",
+  "storagesync": "Storage",
```

Comment:
We sure we want this to be under Storage? It would be ideal if each sdk service folder had its own unique name.

Reply:
Umm we don't know what the correct answer is though- for JS it has a different service folder and for python it doesn't. And not sure what exact Azure service it should map to.

Reply:
cc - @scbedd